src/style/_fonts.ts

```diff
@@ -1,5 +1,5 @@
 export const fontFamilyString = [
-  '"Alef"',
+  'Rubik',
```

Comment:
are we sure Rubik without "" is enough, does it works locally?

Reply:
yes , it works